ajax/libs/react-native-elements/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "react-native-elements",
+  "version": "0.11.2",
+  "description": "React Native Elements & UI Toolkit",
+  "filename": "bundle.js",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/dabit3/React-Native-Elements.git"
```

Comment:
@PeterDaveHello could you please tell me that should I use the same as the upstream or use `git+https//github.com/dabit3/React-Native-Elements.git`
or 
`git+https://github.com/react-native-training/react-native-elements.git`